hal-sim/hal_impl/functions.py

```diff
@@ -1044,6 +1055,19 @@ def observeUserProgramTeleop():
 def observeUserProgramTest():
     hal_data['user_program_state'] = 'test'
 
+def releaseDSMutex():
```

Comment:
This is a bit strange, why isn't there a corresponding acquire function? What is the original function supposed to do (did you read the original source?)

Reply:
Couldn't tell you...